ResumeParseAndNormalize.md

```diff
@@ -348,6 +348,10 @@ Response Structure
 }
 ```
 
+PII Scrubbing
+-----------
+Additionally, the service makes a best attempt at scrubbing Personally Identifiable Information i.e. Social Security Numbers and Driver's License Numbers, from the raw and clean resume fields. Anything found to resemble a SSN or Driver's License Number in the resume text will be replaced with either: "***Social Security Number Removed***" or "***Driver's License Number Removed***". False positives do occur but the scrubbing occurs post parsing and the fields that may be scrubbed are still extracted and returned in their respective fields.
```

Comment:
the comma in the first sentence shouldn't be there, and the "i.e. [...]" bit should probably be in parentheses.

don't we scrub resume HTML too? we should, if not. that also contains the full resume text.

don't we also scrub date of birth unless you use the optional param to return it?

I don't understand the last sentence. it sounds like you're saying "we scrub SSN and DL from the resume text fields, but we leave the SSN and DL fields intact" which 1) would be very silly and 2) is wrong because we don't actually have fields for those things.